### PR TITLE
Adding get for singular endpoint

### PR DESCRIPTION
**Pull request by matzew**
## Proposed Changes

- updating the `GET /endpoints/123` data sample on the spec

**Release Note**

```release-note

```

### CODE REVIEW

File: registry/spec.md [hunk 1]
```diff
@@ -682,8 +682,14 @@ Response:
 HTTP/1.1 ...
 Content-Type: application/json; charset=utf-8
 Content-Length: nnnn
+{
```
Comment (duglin) [inline registry/spec.md:685]: Can you add a blank line before this line?

**Comment by duglin** [comment]
minor comment, otherwise LGTM
thanks!

**Comment by matzew** [comment]
@duglin updated

**Comment by duglin** [comment]
thanks!

File: registry/spec.md (hunk 1 above)
Comment (jskeet) [reply registry/spec.md:686]: There's already a blank line between the headers and the body... is there a reason we'd need another?

**Comment by duglin** [comment]
Approved on the 3/2 call

File: registry/spec.md (hunk 1 above)
Comment (duglin) [reply registry/spec.md:686]: the comment is old. he added the line

**Pull request merged**